TFEngine.py

```diff
@@ -276,7 +276,8 @@ def _collect_eval_info(self, fetches_results):
     """
     # See see self._get_fetches_dict() for the keys.
     # keys are e.g. "cost:output", "error:output" or "loss".
-    keys = [k for k in fetches_results.keys() if k.startswith("cost:") or k.startswith("error:") or k == "loss"]
+    keys = [k for k in fetches_results.keys() if k.startswith("cost:") or k.startswith("error:") or
+            k.startswith("loss:") or k == "loss"]
```

Comment:
The formatting is confusing. `if ...` should be on the same line with all the conditions (all the `or`). If the line becomes too long, make the newline before the `if`.

Reply:
Please resolve this thread if it is resolved (or outdated).